src/Controller/CRUDController.php

```diff
@@ -881,7 +891,10 @@ public function aclAction(Request $request): Response
 
     public function getRequest(): Request
```

Comment:
Are we sure there is always a request or should I add an exception if there is not ?

Why is this method public  BTW ?

Reply:
do we use this method? Controller methods can include the Request parameter directly and remove this.

Reply:
Including the Request parameter would add a BC-break to all the action which require the $request.
We could create an issue about this. WDYT ?

Reply:
I saw some code from master. The request is already on all actions, but it is not passed to all method that need it. I think the BC break will be really low.